### PR TITLE
Adding Abby's coordinates

### PR DESCRIPTION
**Pull request by Abigailfranco6**
Adding Abby's location

### CODE REVIEW

**Comment by brianamarie** [comment]
@bdfy076 Please do not close other people's issues or pull requests. They might want to continue contributing! The GitHub Training team will go through periodically to make sure that the repository is kept tidy. 😄

File: _pins/AbbyG.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: AbbyG
+latitude:47.5321256
```
Comment (hectorsector) [inline _pins/AbbyG.json]: Great work so far @Abigailfranco6! Just make sure you insert a space between the `:` and your latitude.

Comment (taketherose09) [inline _pins/AbbyG.json]: Practice makes perfect

**Comment by brianamarie** [comment]
Hi @Abigailfranco6! I'm going to go ahead and make the changes now.

**Comment by brianamarie** [comment]
@Abigailfranco6 I'm going to merge now. Nice work!

**Pull request merged**